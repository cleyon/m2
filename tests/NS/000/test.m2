@define I 42
BEFORE I=@I@
@for I 1 5
INSIDE I=@I@
@next I
AFTER I=@I@
