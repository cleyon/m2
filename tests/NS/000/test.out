BEFORE I=42
INSIDE I=1
INSIDE I=2
INSIDE I=3
INSIDE I=4
INSIDE I=5
AFTER I=42
